tests/Testcontainers.DockerCompose.Tests/DockerComposeTest.cs

```diff
@@ -1,102 +1,54 @@
-namespace Testcontainers.DockerCompose;
+﻿namespace Testcontainers.DockerCompose;
 
-public abstract class DockerComposeTest : IAsyncLifetime
+public abstract class DockerComposeContainerTest : IAsyncLifetime
 {
     private readonly DockerComposeContainer _dockerComposeContainer;
 
-    protected DockerComposeTest(DockerComposeContainer dockerComposeContainer)
+    private DockerComposeContainerTest(DockerComposeContainer dockerComposeContainer)
     {
         _dockerComposeContainer = dockerComposeContainer;
     }
-    
+
     public Task InitializeAsync()
     {
         return _dockerComposeContainer.StartAsync();
     }
-    
-    public async Task DisposeAsync()
+
+    public Task DisposeAsync()
     {
-        await _dockerComposeContainer.StopAsync();
-        await _dockerComposeContainer.DisposeAsync().AsTask();
+        return Task.CompletedTask;
+        return _dockerComposeContainer.DisposeAsync().AsTask();
     }
-    
+
     [Fact]
     [Trait(nameof(DockerCli.DockerPlatform), nameof(DockerCli.DockerPlatform.Linux))]
     public void ContainerStartedSuccessfully()
     {
+        // TODO: How do we test that the Compose configuration is actually working? How do we access services?
```

Comment:
How can we actually test if the configuration is running? How do we obtain the endpoint of a service, such as the actual hostname and port?

Reply:
To fully support tests with services or containers, we need to introduce an additional abstraction, like [listChildContainers ](https://github.com/testcontainers/testcontainers-java/blob/5969b0f393b724c9ed2147272d87a24c8d9efe1c/core/src/main/java/org/testcontainers/containers/ComposeDelegate.java#L264) in Java. At the moment I would limit myself to checking availability using healthchecks in the file itself.